tests/python/unittest/test_operator.py

```diff
@@ -2131,6 +2131,75 @@ def test_init():
     exec1.forward()
     assert_allclose(exec1.outputs[0].asnumpy(), np.zeros((3,4)))
 
+def test_take():
+    data = mx.sym.Variable('data')
+    idx = mx.sym.Variable('idx')
+    idx = mx.sym.BlockGrad(idx)
+
+    result = mx.sym.take(idx=idx, data=data)
+
+    exe = result.simple_bind(default_context(), data=(10, 5), idx=(4, ))
```

Comment:
@piiswrong The `default_context()` here should make the GPU version automatically tested via `nosetests test_operator_gpu.py`. May double check to make sure.